cmd/collector/app/builder/builder_flags.go

```diff
@@ -23,13 +23,16 @@ import (
 )
 
 const (
-	collectorQueueSize           = "collector.queue-size"
-	collectorNumWorkers          = "collector.num-workers"
-	collectorWriteCacheTTL       = "collector.write-cache-ttl"
-	collectorPort                = "collector.port"
-	collectorHTTPPort            = "collector.http-port"
-	collectorZipkinHTTPort       = "collector.zipkin.http-port"
-	collectorHealthCheckHTTPPort = "collector.health-check-http-port"
+	collectorQueueSize     = "collector.queue-size"
+	collectorNumWorkers    = "collector.num-workers"
+	collectorWriteCacheTTL = "collector.write-cache-ttl"
+	collectorPort          = "collector.port"
+	collectorHTTPPort      = "collector.http-port"
+	collectorZipkinHTTPort = "collector.zipkin.http-port"
+	// CollectorHealthCheckHTTPPort is the flag name for health check
+	CollectorHealthCheckHTTPPort = "collector.health-check-http-port"
```

Comment:
I think a better approach would be to have a single global flag for all binaries, similar to how we do log levels or metrics.

Reply:
Questions,
- Does this mean we need to take that flag out from collector flags and make a global flag?
>  similar to how we do log levels or metrics.
- Are you talking about BasicOptions(cmd/builder/builder_options.go)? How about SharedFlags(cmd/flags/..)?

Reply:
1 - yes
2 - I meant shared flags, the BasicOption is independent of cli flags